pkg/poller/prometheus.go

```diff
@@ -24,6 +25,9 @@ func (u *UnifiPoller) RunPrometheus() error {
 		ReportErrors: true, // XXX: Does this need to be configurable?
 	}))
 
+	version.Version = Version
+	prometheus.MustRegister(version.NewCollector("unifipoller"))
```

Comment:
thinking about this too. I think, lets do `strings.Replace(appName, "-")` instead of `"unifipoller"`

Reply:
I'd rather not, as there are a bunch of other illegal characters. Better is to use a defined `metricNamespace` that's shared with the Desc() calls.